components/organisms/EditResolverModalContent.tsx

```diff
@@ -0,0 +1,149 @@
+import { chain } from "@/lib/wallet/wallet-config";
+import { setResolver } from "@ensdomains/ensjs/wallet";
+import { Button, Input } from "@ensdomains/thorin";
+import { useState } from "react";
+import toast from "react-hot-toast";
+import { Address, createWalletClient, custom, isAddress } from "viem";
+
+interface EditResolverModalContentProps {
+  name: string;
+  currentResolverAddress: Address;
+  closeModal: () => void;
+  onRecordsEdited?: () => void;
+}
+
+export const EditResolverModalContent = ({
+  currentResolverAddress,
+  closeModal,
+  name,
+}: EditResolverModalContentProps) => {
+  const [resolverAddress, setResolverAddress] = useState<string>(
+    currentResolverAddress
+  );
+  const [transactionHash, setTransactionHash] = useState("");
+  const [transactionSuccess, setTransactionSuccess] = useState(false);
+  const [isLoading, setIsloading] = useState(false);
+
+  const handleSaveAction = async () => {
+    // Create a wallet client for sending transactions to the blockchain
+    const walletClient = createWalletClient({
+      chain: chain,
+      transport: custom(window.ethereum),
+    });
+
+    const [address] = await walletClient.getAddresses();
+
+    if (isAddress(resolverAddress)) {
+      setIsloading(true);
+      try {
+        const hash =
+          address &&
+          (await setResolver(walletClient, {
+            name: name,
+            contract: "nameWrapper",
+            resolverAddress: resolverAddress,
+            account: address,
+          }));
+        setTransactionHash(hash);
+        setTransactionSuccess(true);
+      } catch {
+        () => {
+          toast.error("an error occured");
+        };
+      }
+      setIsloading(false);
+    } else {
+      toast.error("Your resolver must be an address");
+    }
+  };
+
+  return (
+    <div className="w-[480px] border rounded-xl overflow-hidden">
+      <div className="py-5 px-6 flex justify-between w-full bg-gray-50 border-b font-semibold text-black">
+        Edit Resolver
+      </div>
+      <div className="bg-white text-black border-b border-gray-200 p-6 flex flex-col gap-4">
+        {transactionSuccess ? (
+          <>
+            <p className="text-[72px]"> 🎉</p>
```

Comment:
Hey this is the third time I replace a text-[72px]! 😁

```suggestion
            <p className="text-7xl"> 🎉</p>
```